core-bundle/config/services.yaml

```
@@ -1407,6 +1413,12 @@ services:
         arguments:
             - '@request_stack'
 
+    contao.webauthn.user_entity_guesser:
+        class: Webauthn\Bundle\Security\Guesser\CurrentUserEntityGuesser
```

```suggestion
        class: Webauthn\Bundle\Security\Guesser\UserEntityGuesser
```

At least according to the service ID, the class should be named `UserEntityGuesser`. I wonder why our service linter did not find this 🤔

---

That won't work. `Webauthn\Bundle\Security\Guesser\UserEntityGuesser` is an interface, not a class. And the naming is imho correct, because `contao.webauthn.user_entity_guesser` is a `UserEntityGuesser`.